### PR TITLE
fix: remove unused deps in `.proto` files

### PR DESCRIPTION
**Pull request by lesterli**
## Summary

This PR removes unused dependencies in `.proto` files, addressing the protobuf generation issue outlined in [babylon-contract/issues/101](https://github.com/babylonlabs-io/babylon-contract/issues/101).

## Test Plan

```
make proto-lint
```

### CODE REVIEW

**Review by maurolacy — APPROVED** [review]
LGTM.

Adding the `proto-lint` target as part of protobuf generation or in CI tests looks like a good idea.

In particular since `buf` is failing on the Rust side because of these unused messages (see https://github.com/babylonlabs-io/babylon-contract/issues/101). And in general because it's good practice.

**Comment by lesterli** [comment]
> Adding the `proto-lint` target as part of protobuf generation or in CI tests looks like a good idea.

yes, added

File: Makefile [hunk 1]
```diff
@@ -419,7 +419,7 @@ protoImage=$(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace $(pro
 
 proto-all: proto-gen proto-swagger-gen ## Generate all protobuf related files
 
-proto-gen: ## Generate protobuf files
+proto-gen: proto-lint ## Generate protobuf files
```
Comment (maurolacy) [inline Makefile:422]: 👍🏼

**Comment by maurolacy** [comment]
Yes, we can backport this to main branch as well.

**Pull request merged**